CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpC
 * Raised minimum PHP version to 5.6.
 * **BC break:** Removed the `Interface` suffix from all interfaces.
 * **BC break:** Renamed `HashGenerator` to `DefaultHashGenerator`.
+* **BC break:** The `ResponseTagger` now expects an instance of
+  `TagHeaderFormatterInterface` instead of `TagCapable` as first argument.
```

Comment:
version 1 had no ResponseTagger nor the TagCapable interface, so this is not really a BC break. lets add a header 2.0.0-beta3 below, above 1.4.2 and say this there. and instead edit the Tagging section in the 2.0 change notes.